File: cpp/example/src/ContainsDuplicate/containsDuplicate.cpp
#include "containsDuplicate.h"

bool containsDuplicate::naive(vector<int>& nums){
    set<int> d;
    for (int i:nums){
        if (d.find(i)!=d.end()) return true;
        d.insert(i);
    }
    return false;
}

File: cs/sample/Sample/ContainsDuplicate/ContainsDuplicate.cs
using System;
using System.Collections.Generic;

namespace Sample.ContainsDuplicate{
    public class containsDuplicate{
        public static bool naive(int[] nums){
            HashSet<int> d = new HashSet<int>();
            foreach (int i in nums){
                if (d.Contains(i)) return true;
                d.Add(i);
            }
            return false;
        }
    }
}
